code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1425,24 +1425,44 @@ GLOBAL_LIST_EMPTY(features_by_species)
 		switch(H.nutrition)
 			if(NUTRITION_LEVEL_FULL to INFINITY)
 				H.throw_alert("nutrition", /atom/movable/screen/alert/fat)
```

Comment:
Make sure that being fat removes the moodspeed modifier too. Also remove line 1414 which adds slowdown if mood is disabled in the config since hunger being tied to slowdown becomes the default